Overzicht portfolio, Student post, Aspecten aanmaken & Aspecten ratings aanmaken

lucius02 opened the pull request.

lucius02 committed:
CArd component made, border not right yet.

lucius02 committed:
posted test card made

lucius02 committed:
problem with sorting on date

lucius02 committed:
extra test cards and footer standard at the bottom of the page

lucius02 committed:
orderby solved

lucius02 committed:
unnecessary file

spark-156 commented on frontend/components/CardCom.vue:
```diff
@@ -0,0 +1,90 @@
+  data () {
+    return {
+      lists: [
+        {
+          id: 1,
+          title: 'test1',
+          date: '11-03-2022',
+          text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhonc ...
+        },
+        {
+          id: 3,
+          title: 'test3',
+          date: '07-02-2022',
+          text: 'tLorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhon ...
+        },
+        {
+          id: 2,
+          title: 'test2',
+          date: '14-03-2022',
+          text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhonc ...
+        },
+        {
+          id: 5,
+          title: 'test5',
+          date: '15-05-2022',
+          text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhonc ...
+        },
+        {
+          id: 6,
+          title: 'test6',
+          date: '07-06-2022',
+          text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhonc ...
+        },
+        {
+          id: 4,
+          title: 'test4',
+          date: '14-04-2022',
+          text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhonc ...
+        }
+      ]
```
![image](https://user-images.githubusercontent.com/57094525/158551694-10828efc-e08c-4501-99c6-4175daab2333.png)
Dit is alle data die je in een post terug krijgt. kan je het een klein beetje aanpassen zodat alles goed getoond wordt met dezelfde benaming?

spark-156 commented on frontend/components/CardCom.vue:
```diff
@@ -0,0 +1,90 @@
+
+<style scoped>
+  .postcard{
+      width: 90vw;
```
max-width?

spark-156 replied:
min width ook

spark-156 commented on frontend/components/CardCom.vue:
```diff
@@ -0,0 +1,90 @@
+<style scoped>
+  .postcard{
+      width: 90vw;
+      height: 25vh;
```
max height?

spark-156 replied:
min height ook

spark-156 commented on frontend/components/FooterCom.vue:
```diff
@@ -1,20 +1,13 @@
-    <v-app-bar color="#0079CF" />
-  </div>
+  <v-footer
+    color="#0079CF"
```
Gebruik een config file zodat er een centrale plek is waar je al je kleuren kan beheren

spark-156 commented on frontend/components/CardCom.vue:
```diff
@@ -0,0 +1,90 @@
+      margin-bottom: 2vw;
+  }
+  .header{
+      background-color: #0079CF;
```
Gebruik een config file zodat er een centrale plek is waar je al je kleuren kan beheren

spark-156 commented on frontend/components/CardCom.vue:
```diff
@@ -0,0 +1,90 @@
+import _ from 'lodash'
+
+export default {
+  name: 'CardCom',
```
waarom kort je het af? waarom niet CardComponent

spark-156 replied:
Niet iets resolven zonder dat je het besproken hebt aub

spark-156 commented on frontend/components/HeaderCom.vue:
```diff
@@ -67,5 +67,9 @@ export default {
     position: sticky;
     z-index: 2;
     border-radius: 0;
+    background-color: #0079CF;
```
Gebruik een config file zodat er een centrale plek is waar je al je kleuren kan beheren

spark-156 commented on frontend/components/SearchSortBar.vue:
```diff
@@ -0,0 +1,68 @@
+      return this.keys.filter(key => key !== 'Title')
+    }
+  }
+}
```
Er moet nog een api call komen voor de backend

spark-156 committed:
Merge remote-tracking branch 'origin/FEEBY-51-lokale-dev-environment-koppelen-aan-lti' into Overzicht-Portfolio

spark-156 committed:
Merge branch 'Overzicht-Portfolio' of github.com:FeebyOpenICT/Feeby into Overzicht-Portfolio

spark-156 committed:
frontend hot reload possible fix

lucius02 committed:
updatev gitignore

lucius02 committed:
Merge branch 'Overzicht-Portfolio' of https://github.com/FeebyOpenICT/Feeby into Overzicht-Portfolio

lucius02 committed:
style changed, color changed and props tried

spark-156 committed:
Merge branch 'dev' into Overzicht-Portfolio

lucius02 committed:
logo is now / link, icon is white

lucius02 committed:
Merge branch 'Overzicht-Portfolio' of https://github.com/FeebyOpenICT/Feeby into Overzicht-Portfolio

timrebergen committed:
method Post voor Docker update in CreatePost 17-03 - 14:35PM

Een apart JS bestand met een POST request voor de post aanmaken

timrebergen committed:
Merge branch 'Student-Post' of https://github.com/FeebyOpenICT/Feeby into Student-Post

lucius02 committed:
nuxtlink added

timrebergen committed:
Post maken aanpassingen

spark-156 committed:
Merge branch 'Student-Post' into Overzicht-Portfolio

lucius02 committed:
card pros removed

timrebergen committed:
Update Post

spark-156 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Overzicht van portfolio](https://trello.com/c/nnjCxZSd/121-overzicht-van-portfolio)

timrebergen committed:
Nieuwe create post 21-3- 10:04AM

lucius02 committed:
docker live-reload succes, css updated for card and header

lucius02 committed:
Merge branch 'dev' into Overzicht-Portfolio

lucius02 committed:
Merge branch 'Student-Post' into Overzicht-Portfolio

lucius02 committed:
axiosInstance made, axiosinstance used in cardcom

lucius02 committed:
fetch works

timrebergen committed:
Student post changes 24-3 9:51AM

Student post changes 24-3 9:51AM

timrebergen committed:
Merge remote-tracking branch 'origin/Overzicht-Portfolio' into Student-Post

lucius02 committed:
footer completed

lucius02 committed:
Merge branch 'Aspecten-ratings-aanmaken---admin-pagina' into aspect-rating-aanamaken

spark-156 committed:
Merge branch 'Overzicht-Portfolio' of github.com:FeebyOpenICT/Feeby into Overzicht-Portfolio

lucius02 committed:
aspect aanmaken

timrebergen committed:
ophalen_cards_post

Het ophalen van de kaartjes vanuit de back-end.

lucius02 committed:
Merge branch 'Student-Post' into aspect-rating-aanamaken

timrebergen committed:
Revert node modules

timrebergen committed:
Merge branch 'dev' into Student-Post

timrebergen committed:
Revert AXIOS node Modules

lucius02 committed:
aspect rating post added, node_modules on wrong place deketed

spark-156 reviewed (changes requested):
er is nog geen koppeling naar de backend

timrebergen committed:
Add button toegevoegd als Component

Er is een component aangemaakt waarin een 'Add Post' button staat. Deze component kan je gebruiken op elke pagina waarop je wilt linken naar de post maken pagina. De routing linkt werkt ook naar behoren.

timrebergen committed:
Foutmelding material design compile opgelost

lucius02 committed:
aspect rating aanmaken done, style not doen

lucius02 committed:
theme from aspectratingaanmaken changed to light

timrebergen committed:
Live reload fix

timrebergen committed:
Merge branch 'aspect-rating-aanamaken' into Student-Post

lucius02 committed:
getting ratings and added post

timrebergen committed:
Overlay fix Aspectmaken

lucius02 committed:
Merge branch 'Student-Post' into aspect-rating-aanamaken

lucius02 committed:
details from aspect

lucius02 committed:
Merge branch 'aspect-rating-aanamaken' into Overzicht-Portfolio

spark-156 commented:
Aub niet andere branches in jouw branch morgen, nu kan ik hem niet in dev morgen totdat alles klaar is.
